apps/web/src/features/organizations/components/MembersList/RemoveMemberModal.tsx

```diff
@@ -0,0 +1,50 @@
+import ModalDialog from '@/components/common/ModalDialog'
+import { DialogContent, DialogActions, Button, Typography } from '@mui/material'
+import type { UserOrganizationUser } from '@safe-global/store/gateway/AUTO_GENERATED/organizations'
+import { useUserOrganizationsRemoveUserV1Mutation } from '@safe-global/store/gateway/AUTO_GENERATED/organizations'
+import { useCurrentOrgId } from '@/features/organizations/hooks/useCurrentOrgId'
+import { useState } from 'react'
+
+const RemoveMemberDialog = ({ member, handleClose }: { member: UserOrganizationUser; handleClose: () => void }) => {
+  const orgId = useCurrentOrgId()
+  const [deleteMember] = useUserOrganizationsRemoveUserV1Mutation()
+  const [errorMessage, setErrorMessage] = useState<string>('')
+
+  const handleConfirm = async () => {
+    setErrorMessage('')
+    try {
+      const { error } = await deleteMember({ orgId: Number(orgId), userId: member.id })
+
+      if (error) {
+        throw error
+      }
+    } catch (e) {
+      setErrorMessage('An unexpected error occurred while removing the member.')
+    }
+  }
+
+  return (
+    <ModalDialog open onClose={handleClose} dialogTitle="Remove member" hideChainIndicator>
+      <DialogContent sx={{ p: '24px !important' }}>
+        {/* TODO: use name here instead of ID as soon as it's available */}
+        <Typography>Are you sure you want to remove {`${member.id}`} from this organization?</Typography>
```

Comment:
If you pull in the epic branch the name property should now be available on the `member` object.

nit: I would make the name bold to have it more focused from the rest of the sentence.